kcptun.rb

```diff
@@ -0,0 +1,72 @@
+class Kcptun < Formula
+  desc "extremely simple & fast udp tunnel based on kcp protocol"
+  homepage "https://github.com/xtaci/kcptun"
+  url "https://github.com/xtaci/kcptun.git",
+    :tag => "v20160701", :revision => "13e980cdffb02e0b2bc793366e1314e663f549fc"
+
+  head "https://github.com/xtaci/kcptun.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath/".kcptun-gopath"
+    mkdir_p buildpath/".kcptun-gopath/src/github.com/xtaci"
+    cp_r cached_download, buildpath/".kcptun-gopath/src/github.com/xtaci/kcptun"
+    ENV.append_path "PATH", "#{ENV["GOPATH"]}/bin"
+    cd buildpath/".kcptun-gopath/src/github.com/xtaci/kcptun"
+    system "go", "get", "-v", "./client"
+    system "go", "build", "-ldflags", "-X main.VERSION=20160701 -s -w", "-o", "kcptun_client", "github.com/xtaci/kcptun/client"
```

Comment:
Probably want to use `#{version}` instead of hardcoding it, and please wrap this to ~80 characters.